src/parcels/mount-parcel.js

```diff
@@ -66,9 +66,8 @@ export function mountParcel(config, customProps) {
         })
         .catch(err => {
           parcel.status = SKIP_BECAUSE_BROKEN;
-          const transformedErr = transformErr(err, parcel);
```

Comment:
this was causing an extra thing being added to the error message that is already being added by toUnmountPromise